src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const populationSpans = document.querySelectorAll('.population');
+
+let totalPopulation = 0;
+let count = 0;
+
+populationSpans.forEach((span) => {
+  const populationText = span.textContent;
+
+  const populationNumber = Number(populationText.split(',').join(''));
```

Comment:
The conversion of `populationText` to a number assumes that the text is always a valid number format. If there are any non-numeric characters (other than commas), this will result in `NaN`. Consider adding error handling or validation to ensure the text is a valid number before conversion.